src/webserver.py

```diff
@@ -59,6 +59,9 @@ def _config(self) -> str:
         config = f"""\
 bind = ['0.0.0.0:{self._charm_state.flask_port}']
 chdir = {repr(str(self._charm_state.flask_dir.absolute()))}
+accesslog = {repr(str(self._charm_state.flask_access_log.absolute()))}
```

Comment:
nitpick: access_log/error_log?

Reply:
Those are configuration fields required by gunicorn, we can't change that unfortunately.